recipes/gpmdp.mdwn

```diff
@@ -0,0 +1,21 @@
+# Google Play Music Desktop Player watcher
+
+This widget can be used to display the current track information and state of [Google Play Music Desktop Player](https://www.googleplaymusicdesktopplayer.com/).
+
+1. Download `gpmdp.lua` [here](https://awesomewm.org/recipes/gpmdp.lua)
+2. Add your customization to `gpmdp.lua` starting at line 87
+    > for a in depth example, see [here](https://github.com/gregflynn/dotsanity/blob/master/awesome/widgets/gpmdp.lua)
+
+3. Import `gpmdp.lua` into your `rc.lua`
+    ```lua
+    local gpmdp = require("path/to/gpmdp.lua")
+    ```
+4. Add the widget to your wibar
+    ```lua
+    s.mywibox:setup {
+        layout = wibox.layout.align.horizontal,
+        {
+            layout = wibox.layout.fixed.horizontal,
+            require("widgets/gpmdp").widget
```

Comment:
```diff
-            require("widgets/gpmdp").widget
+            gpmdp.widget,
+            -- [...]
```

Since you imported it in the previous step.